### PR TITLE
week1_ABilgundi

### PR DESCRIPTION
**Pull request by adityabilgundi-1738**

### CODE REVIEW

File: week1/armstrong.rb [hunk 1]
```diff
@@ -9,5 +9,26 @@
...
+	    digits+=1
+	    temp=temp/10
+    end
+
+    temp = num
+
+    while (temp > 0)
+        sum += (temp%10)**digits
+        temp = temp/10
+    end
```
Comment (raosush) [inline week1/armstrong.rb:27]: You can very easily find the digits of a number:
```ruby
array_of_digits = n.digits
```

**Review by raosush — APPROVED** [review]
Good job! Assignment 1 completed 😃

**Comment by raosush** [comment]
Good job, closing the pull request since changes have been approved

File: week1/armstrong.rb (hunk 1 above)
Comment (adityabilgundi-1738) [reply week1/armstrong.rb:27]: Ah yes! Thanks!

**Pull request closed without merging**